src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -294,22 +308,19 @@ export default class ComparisonGroupManager extends React.Component<IComparisonG
                                 placeholder="Enter a name for your new group"
                                 value={this.inputGroupName}
                                 onChange={this.onChangeInputGroupName}
+                                onKeyPress={
+                                    async(event) => {
+                                        if(event.key == 'Enter' && this.inputGroupName.length && allGroupNames && (allGroupNames.indexOf(this.inputGroupName) === -1)){
```

Comment:
this is missing one entry from the `disabled` logic on line 324. 

Let's reuse that logic:

```
@computed get submitNewGroupDisabled() {
     const selectedSamples = this.props.store.selectedSamples.isComplete ? this.props.store.selectedSamples.result :  undefined;
     return !selectedSamples || this.inputGroupName.length === 0 || !allGroupNames || (allGroupNames.indexOf(this.inputGroupName) > -1);
}
```

put that in `disabled` on line 324, and use it here on line 313.